src/thunderbird_accounts/authentication/management/commands/make_superuser.py

```diff
@@ -0,0 +1,24 @@
+from django.core.management.base import BaseCommand, CommandError
+from thunderbird_accounts.authentication.models import User
+
+
+class Command(BaseCommand):
```

Comment:
I think we can just use https://docs.djangoproject.com/en/5.1/ref/django-admin/#createsuperuser with the arguments. Unless it errors out on existing user.

Reply:
Just to make sure I'm clear, you're suggesting the following, yes?
* delete the custom command
* change the text to show using the `manage.py` command

Reply:
Yep!

Reply:
Ah - I just remembered why I created the custom command: it's if they already tried to log in via FXA but got `401 Unauthorized`. 

I already have instructions to use the `manage.py createsuperuser`, but you make a good point about adding the `--email <fxa_email_address>`

I'm going to keep the custom command and the instructions so someone can quickly fix the `401 Unauthorized` situation.